engine/apps/twilioapp/twilio_client.py

```diff
@@ -149,6 +157,18 @@ def make_call(self, message, to):
                 status_callback_method="POST",
             )
         except TwilioRestException as e:
+            # If status callback is not valid and not accessible from public url then trying to make call without it
+            # https://www.twilio.com/docs/api/errors/21609
+            if e.code == 21609:
+                logger.warning("twilio_client.make_call: Twilio error 21609. Status Callback is not public url")
+                return self.twilio_api_client.calls.create(
+                    url=url,
+                    to=to,
+                    from_=self.twilio_number,
+                    method="GET",
+                    status_callback_method="POST",
```

Comment:
Do we need to set `status_callback_method` here if we don't specify `status_callback`?